### PR TITLE
LRDOCS-8424 - Update Workflow - Commerce 

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md [hunk 1]
```diff
@@ -6,6 +6,35 @@
 
 Once the buyer has submitted his cart for pre-approval, or placed an order for seller-approval, the order can be approved or rejected by the appropriate user as follows:
 
+1. Click the _Personal_ menu &rarr; _Notifications_.
+1. Click on the order workflow notification.
+1. Click the _3-dot icon_ in the upper-right corner → *Assign to Me*.
+
+   ![Review](./approving-or-rejecting-orders-in-order-workflows/images/03.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:13]: Please update the image alt text to be a complete sentence.

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md [hunk 2]
```diff
@@ -6,6 +6,35 @@
...
+
+    ```note::
+       Alternatively, click `Assign to...` and select another user to review the Commerce Order.
+    ```
+
+1. Add a comment (optionally) and click *Done*.
+1. Click the _3-dot icon_ in the upper-right corner → *Approve* (or *Reject*).
+
+   ![Review Drop Down](./approving-or-rejecting-orders-in-order-workflows/images/04.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:22]: Please update the image alt text to be a complete sentence.

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md [hunk 3]
```diff
@@ -6,6 +6,35 @@
...
+
+1. Add a comment (optionally) and click *Done*.
+
+   ![Adding a Comment](./approving-or-rejecting-orders-in-order-workflows/images/05.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:26]: Please update the image alt text to be a complete sentence.

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md [hunk 4]
```diff
@@ -6,6 +6,35 @@
...
+
+    The reviewed order is then displayed in _My Workflow Tasks_.
+
+   ![Order Approved](./approving-or-rejecting-orders-in-order-workflows/images/06.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:30]: Please update the image alt text to be a complete sentence.

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/enabling-or-disabling-order-workflows.md [hunk 5]
```diff
@@ -8,6 +8,20 @@ This article documents how to enable and disable Order Workflows.
 
 To enable an approval workflow:
 
+1. Go to the _Global Applications_ menu &rarr; _Commerce_ &rarr; _Channels_.
+1. Select the desired channel (for example, _Sahara.com_). (If you had used an accelerator to create a site, there is an associated channel created by default.)
+1. Click on the _General_ tab.
+1. Select _Single Approver (Version 1)_ from the Buyer Order Approval Workflow dropdown menu.
+
+    ![Enable Buyer Order Approval workflow.](./enabling-or-disabling-order-workflows/images/06.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/enabling-or-disabling-order-workflows.md:16]: Please update the image alt text to be a complete sentence.

**Comment by jrhoun** [comment]
Please re-send once you've updated! Thanks.

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:13]: Updated

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md (hunk 3 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:26]: Updated

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md (hunk 4 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:30]: Updated

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/enabling-or-disabling-order-workflows.md (hunk 5 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/enabling-or-disabling-order-workflows.md:16]: Updated

File: docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/order-workflows/approving-or-rejecting-orders-in-order-workflows.md:22]: Updated

**Pull request closed without merging**